apps/cookbook/src/app/showcase/slides-showcase/slides-showcase.component.html

```diff
@@ -101,6 +101,31 @@ <h2>Slots</h2>
   to the navigation controls (large screens only).
 </p>
 
+<h2>Inherit height</h2>
+
+<p>
+  If the swiper-slides has variable height, you can add the directive
+  <code>slideStretchHeight</code>
+  to kirby slides, to make all slides equal height as seen in the example below.
+</p>
```

Comment:
Consider leaving swiper out of the wording to avoid confusion:
```suggestion
<p>
  If the slides have variable height, you can add the directive
  <code>slideStretchHeight</code>
  to each slide to make them equal height as seen in the example below.
</p>
```